adapters/puppeteer/src/index.ts

```diff
@@ -102,10 +102,9 @@ export const pupUniDriver = (
     };
 
     const clearValue = async() => {
-        const { element, page } = await elem();
-        await page.evaluate((element) => {
-            element.value = '';
-        }, element);
+        const { element } = await elem();
+        await element.click({clickCount: 3});
```

Comment:
why is 3 clicks needed?

Reply:
https://github.com/segmentio/nightmare/issues/810#issuecomment-454993184
Triple click selects all input content